srt.go

```diff
@@ -202,6 +202,7 @@ func parseTextSrt(i string) (o Line) {
 				}
 
 				// Append item
+				s = unescapeWebVTT(s)
```

Comment:
Could you:
- move `webVTTEscaper` and `webVTTUnescaper` to `subtitles.go`
- rename them to `htmlEscaper` and `htmlUnescaper`
- move `escapeWebVTT` and `unescapeWebVTT` to `subtitles.go`
- rename them `escapeHTML` and `unescapeHTML`